{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--8b95b571-ba64-4c00-89a7-6113cb272e0d",
    "created": "2020-05-26T20:29:31.728703Z",
    "modified": "2020-05-26T20:29:31.728703Z",
    "name": "CVE-2017-14503",
    "description": "libarchive 3.3.2 suffers from an out-of-bounds read within lha_read_data_none() in archive_read_support_format_lha.c when extracting a specially crafted lha archive, related to lha_crc16."
}